src/openvpn/ssl.c

```diff
@@ -604,6 +604,13 @@ init_ssl(const struct options *options, struct tls_root_ctx *new_ctx, bool in_ch
 
     tls_clear_error();
 
+#ifdef HAVE_XKEY_PROVIDER
+    if (key_is_external(options))
+    {
+        load_xkey_provider();
```

Comment:
load_xkey_provider is always defined, so either we should always try to load it by removing the ifdef here or also only define load_xkey_provider when HAVE_XKEY_PROVIDER is defined.